airflow/example_dags/example_subdag_operator.py

```diff
@@ -19,44 +19,52 @@
 
 from __future__ import annotations
 
-# [START example_subdag_operator]
-import datetime
-
-from airflow.example_dags.subdags.subdag import subdag
-from airflow.models.dag import DAG
-from airflow.operators.empty import EmptyOperator
-from airflow.operators.subdag import SubDagOperator
-
-DAG_NAME = "example_subdag_operator"
-
-with DAG(
-    dag_id=DAG_NAME,
-    default_args={"retries": 2},
-    start_date=datetime.datetime(2022, 1, 1),
-    schedule="@once",
-    tags=["example"],
-) as dag:
-    start = EmptyOperator(
-        task_id="start",
-    )
+import warnings
 
-    section_1 = SubDagOperator(
-        task_id="section-1",
-        subdag=subdag(DAG_NAME, "section-1", dag.default_args),
+with warnings.catch_warnings():
+    warnings.filterwarnings(
+        "ignore",
+        message=r"This class is deprecated\. Please use `airflow\.utils\.task_group\.TaskGroup`\.",
```

Comment:
I'm not sure, maybe better to use here some condition whether it run in unit tests or not

Reply:
And also fix test DAGs, because right now it doesn't suppress anything, just capture warnings

Reply:
What I can't find where the actual warnings raised in Airflow environment, seems it's depends on deployment and it might be captured in some subprocess as result it do not shown into the any logs (task, scheduler, worker, dag_processor) even without this suppression.

Reply:
with SubDag being long deprecated, that's probably fine to add it here.